foundation-extensions/adobe-experience-platform-assurance/README.md

```diff
@@ -212,21 +239,45 @@ Registering the extension with Core, sends Experience Platform SDK events to an
    ```
 {% endtab %}
 
-{% tab title="iOS" %}
-Registering the extension with Core sends Experience Platform SDK events to an active Project Griffon session. To start using the extension library, you must first register the extension with the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension.
+{% tab title="iOS (AEP 3.x)" %}
+
+To start using the extension library, you must first register the extension with the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension.
```

Comment:
You don't need to make this change yet, but using relative links (as in linking within how it looks within the repo) is preferred over using absolute links (which is linking where it is within the website).